ModsenOnlineStore.Store.Infrastructure/Services/ProductService/ProductService.cs

```diff
@@ -0,0 +1,60 @@
+﻿using AutoMapper;
+using ModsenOnlineStore.Common;
+using ModsenOnlineStore.Store.Application.Interfaces.ProductInterfaces;
+using ModsenOnlineStore.Store.Domain.DTOs.ProductDTOs;
+using ModsenOnlineStore.Store.Domain.Entities;
+
+namespace ModsenOnlineStore.Store.Infrastructure.Services.ProductService
+{
+    public class ProductService : IProductService
```

Comment:
such services should be placed in application layer, infrastructure layer should store logic of interacting with  external objects like db, file system and so on